util/deephash/deephash.go

```diff
@@ -136,7 +137,21 @@ func print(w *bufio.Writer, v reflect.Value, visited map[uintptr]bool, scratch [
 		return acyclic
 	case reflect.Slice, reflect.Array:
 		if v.Type().Elem().Kind() == reflect.Uint8 && v.CanInterface() {
-			fmt.Fprintf(w, "%q", v.Interface())
+			if v.CanAddr() {
+				Len := v.Len()
+				w.Write(strconv.AppendInt(scratch[:0], int64(Len), 10))
+				w.WriteByte(':')
```

Comment:
would it be better to append the `:` to the output of `strconv.AppendInt` and then write the resulting slice, rather than a big write followed by a small write, or is that not going to matter?

For that matter, if we're worried about performance, would it be better to write the length with `encoding/binary` and skip the colon?

Reply:
Changed to using binary throughout.

(removing the human-readable "deepprint" printable output vestiges)